cdk/src/stacks/living-documentation.ts

```diff
@@ -14,6 +14,14 @@ export class LivingDocumentationStack extends NestedStack {
       type: 'String',
     });
 
+    const hostedZoneParamName = new CfnParameter(this, 'HostedZoneParamName', {
```

Comment:
do we need to pass it via CfnParameter? Is there any benefit over passing it via props?

Reply:
True, it doesn't have any benefit. But it felt cleaner to define all parameters for the constructor in a consistent way (cfnparameter) than mixing with props (I think "domain" there uses cfnParameter to pass the unit test). What do you think?